File: Python Code/Daily_Flash_Code_0/Week12/11Apr/71-DailyFlash_Solutions/11_April_Solution_Three/C/prog5.c
#include<stdio.h>
#include<math.h>
int main() {
	float a, b, c;
	scanf("%f %f %f", &a, &b, &c);
	printf("%f %f", (-b + sqrt(b * b - 4 * a * c)/(2 * a)), (-b - sqrt(b * b - 4 * a * c)/(2 * a)));
}	

File: Python Code/Daily_Flash_Code_0/Week 8/11 Mar/44-DailyFlash_Solutions/44-DailyFlash_Solutions/11-Mar/Python/prog5.py
import math
x1 = int(input("x1 : "))
y1 = int(input("y1 : "))
x2 = int(input("x2 : "))
y2 = int(input("y2 : "))
x3 = int(input("x3 : "))
y3 = int(input("y3 : "))
print("AB :", math.sqrt((y1 - y2) * (y1 - y2) + (x1 - x2) * (x1 - x2)), "\nBC :", math.sqrt((y3 - y2) * (y3 - y2) + (x3 - x2) * (x3 - x2)), "\nCA :", math.sqrt((y1 - y3) * (y1 - y3) + (x1 - x3) * (x1 - x3)))
